src/main/java/teammates/storage/sqlsearch/SearchManager.java

```diff
@@ -223,8 +223,8 @@ List<T> convertDocumentToEntities(List<SolrDocument> documents) {
             if (entity == null) {
                 // search engine out of sync as SearchManager may fail to delete documents
                 // the chance is low and it is generally not a big problem
-                String id = (String) document.getFirstValue("id");
-                deleteDocuments(Collections.singletonList(id));
+                // String id = (String) document.getFirstValue("id");
+                // deleteDocuments(Collections.singletonList(id));
```

Comment:
nit: let's explain why this is commented out, can actually reference ur PR explaination